Beocreate2/beo-extensions/ui-settings/ui-settings-client.js

```diff
@@ -1,9 +1,22 @@
+var settings = {
+	screensaver_timeout: 1
+}
+let timer = null;
+//current localbrowser user agent is 'Mozilla/5.0 (X11; Linux armv7l) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.0 Safari/605.1.15'
+let screensaverEnabled = navigator.userAgent.indexOf('(X11; Linux armv7l)') >= 0 && navigator.userAgent.indexOf(' (KHTML, like Gecko) Version/13.0 Safari') >= 0;
```

Comment:
The userAgent could be customized to something very specific on hifiberry-os e.g. "hifiberry-localbrowser", but I'm not sure if there is any other current use of userAgent that would break.